views/js/controllers/module/module_card.js

```diff
@@ -244,28 +244,29 @@ var AdminModuleCard = function () {
                             mainElement = jqElementObj.parents('.' + alteredSelector).first();
                             mainElement.remove();
                         });
-                        BOEvent.emitEvent("Module Uninstalled", "CustomEvent");
+//                        BOEvent.emitEvent("Module Uninstalled", "CustomEvent");
```

Comment:
Why are these commented out?

Reply:
I don't need it on my module, I handle the module count differently

Reply:
It's best not to leave out commented code if it's not needed anymore

Reply:
Ok i'll remove them